src/UIComponents/SaveModel.jsx

```diff
@@ -59,38 +61,40 @@ class SaveModel extends Component {
     var fields = [];
     fields.push({
       id: "potentialUses",
-      text: "Intended Use",
+      text: "Intended Use:",
       description:
         "Describe the problem you think this model could help solve, or one \
         potential app someone could make with this model.",
       placeholder: "Write a brief description."
     });
     fields.push({
       id: "potentialMisuses",
-      text: "Warnings",
+      text: "Warnings:",
       description:
         "Describe any situations where this model could potentially \
         be misused, or any places where bias could potentially show up in the \
         model. Important questions to consider are:",
-      descriptionDetailOne: "Is there enough data to create an accurate model?",
-      descriptionDetailTwo: "Does the data represent all possible users and scenarios?",
+      descriptionDetails: [
+        "Is there enough data to create an accurate model?",
+        "Does the data represent all possible users and scenarios?"
+      ],
       placeholder: "Write a brief description."
     });
 
     return fields;
   };
 
   render() {
-
     const nameField = {
       id: "name",
-      text: "What will you name the model? (required)"
+      text: "Model name:"
```

Comment:
Do we still want to indicate somehow that this is a required field? Or ok to just let users find that out if they hit an error by trying to save without a name?

Reply:
Ooo - good catch. I have a semi-strong opinion that we should keep the (required) part because of how it behaves with the "Save" button on this screen. AI Lab hides the "Save" button until you start typing in the model name (you can see this in the video around 1:57). This is the only time in the tool that this happens - all other times, the button is visible but disabled (you can see this with "Continue" and "Train" buttons earlier in the video). 

I worry that a user will get to that screen and see the Save button is missing, then think they did something wrong, when in reality they just need to start typing in Model Name section for it to appear (this has also happened to me a few times). I think including the "required" label would help cue a user that they need to do something here first, and then they would notice that the Save button appears (which has also happened to me).

(As I wrote this, I started to wonder: why does the Save button behave differently on this panel than the others?)

Reply:
I restored `(required)`.  These concerns are valid.  For what it's worth, we do disable nav buttons in other screens until the user has done something required.

Reply:
I restored `(required)`.  These concerns are valid.  I can change the `Save` button be disabled, rather than hidden, in a followup.